pkg/minikube/registry/drvs/none/none.go

```diff
@@ -61,14 +61,10 @@ func status() registry.State {
 		return registry.State{Running: true, Error: err, Installed: false, Fix: "Install docker", Doc: "https://minikube.sigs.k8s.io/docs/reference/drivers/none/"}
 	}
 
-	u, err := user.Current()
+	_, err = user.Current()
 	if err != nil {
 		return registry.State{Running: true, Error: err, Healthy: false, Doc: "https://minikube.sigs.k8s.io/docs/reference/drivers/none/"}
 	}
 
```

Comment:
This needs to check that either the user is uid==0 or has sudo access. 

Otherwise, the `none` driver may be auto-selected if the user doesn't meet the requirements. It also prevents minikube from breaking if the user doesn't have sudo access. For example, right now this happens:

```
 ./out/minikube start --driver=none          253ms  Wed 28 Oct 2020 11:38:48 AM PDT
😄  minikube v1.14.0 on Debian rodete
✨  Using the none driver based on user configuration
👍  Starting control plane node minikube in cluster minikube
🤹  Running on localhost (CPUs=6, Memory=63978MB, Disk=411674MB) ...
🎉  minikube 1.14.2 is available! Download it: https://github.com/kubernetes/minikube/releases/tag/v1.14.2
💡  To disable this notice, run: 'minikube config set WantUpdateNotification false'

🔥  Deleting "minikube" in none ...
E1028 11:39:05.027805  957984 none.go:188] cleanup incomplete: sudo rm -rf /var/tmp/minikube /etc/kubernetes/manifests /var/lib/minikube: exit status 1
stdout:
stderr:
Sorry, user tstromberg is not allowed to execute '/bin/rm -rf /var/tmp/minikube /etc/kubernetes/manifests /var/lib/minikube' as root on beelzebufo1.sfo.corp.google.com.
🤦  StartHost failed, but will try again: post-start: sudo mkdir (none): sudo mkdir -p /etc/kubernetes/addons /etc/kubernetes/manifests /var/tmp/minikube /var/lib/minikube /var/lib/minikube/certs /var/lib/minikube/images /var/lib/minikube/binaries /tmp/gvisor /usr/share/ca-certificates /etc/ssl/certs: exit status 1
stdout:
stderr:
Sorry, user tstromberg is not allowed to execute '/bin/mkdir -p /etc/kubernetes/addons /etc/kubernetes/manifests /var/tmp/minikube /var/lib/minikube /var/lib/minikube/certs /var/lib/minikube/images /var/lib/minikube/binaries /tmp/gvisor /usr/share/ca-certificates /etc/ssl/certs' as root on beelzebufo1.sfo.corp.google.com.
🤹  Running on localhost (CPUs=6, Memory=63978MB, Disk=411674MB) ...
😿  Failed to start none bare metal machine. Running "minikube delete" may fix it: post-start: sudo mkdir (none): sudo mkdir -p /etc/kubernetes/addons /etc/kubernetes/manifests /var/tmp/minikube /var/lib/minikube /var/lib/minikube/certs /var/lib/minikube/images /var/lib/minikube/binaries /tmp/gvisor /usr/share/ca-certificates /etc/ssl/certs: exit status 1
stdout:
stderr:
Sorry, user tstromberg is not allowed to execute '/bin/mkdir -p /etc/kubernetes/addons /etc/kubernetes/manifests /var/tmp/minikube /var/lib/minikube /var/lib/minikube/certs /var/lib/minikube/images /var/lib/minikube/binaries /tmp/gvisor /usr/share/ca-certificates /etc/ssl/certs' as root on beelzebufo1.sfo.corp.google.com.
❌  Exiting due to GUEST_PROVISION: Failed to start host: post-start: sudo mkdir (none): sudo mkdir -p /etc/kubernetes/addons /etc/kubernetes/manifests /var/tmp/minikube /var/lib/minikube /var/lib/minikube/certs /var/lib/minikube/images /var/lib/minikube/binaries /tmp/gvisor /usr/share/ca-certificates
```

Reply:
Okay, sounds good.